api/pkg/transformer/server/server_test.go

```diff
@@ -314,23 +314,262 @@ func TestServer_PredictHandler_StandardTransformer(t *testing.T) {
 										"driver_feature_1": feastSdk.DoubleVal(1111),
 										"driver_feature_2": feastSdk.DoubleVal(2222),
+										"driver_feature_3": {
+											Val: &feastTypes.Value_StringListVal{
+												StringListVal: &feastTypes.StringList{
+													Val: []string{"A", "B", "C"},
+												},
+											},
+										},
 									},
 									Statuses: map[string]serving.GetOnlineFeaturesResponse_FieldStatus{
 										"driver_id":        serving.GetOnlineFeaturesResponse_PRESENT,
 										"driver_feature_1": serving.GetOnlineFeaturesResponse_PRESENT,
 										"driver_feature_2": serving.GetOnlineFeaturesResponse_PRESENT,
+										"driver_feature_3": serving.GetOnlineFeaturesResponse_PRESENT,
+									},
+								},
+								{
+									Fields: map[string]*feastTypes.Value{
+										"driver_id":        feastSdk.Int64Val(2),
+										"driver_feature_1": feastSdk.DoubleVal(3333),
+										"driver_feature_2": feastSdk.DoubleVal(4444),
+										"driver_feature_3": {
+											Val: &feastTypes.Value_StringListVal{
+												StringListVal: &feastTypes.StringList{
+													Val: []string{"X", "Y", "Z"},
+												},
+											},
+										},
+									},
+									Statuses: map[string]serving.GetOnlineFeaturesResponse_FieldStatus{
+										"driver_id":        serving.GetOnlineFeaturesResponse_PRESENT,
+										"driver_feature_1": serving.GetOnlineFeaturesResponse_PRESENT,
+										"driver_feature_2": serving.GetOnlineFeaturesResponse_PRESENT,
+										"driver_feature_3": serving.GetOnlineFeaturesResponse_PRESENT,
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			rawRequest: request{
+				headers: map[string]string{
+					"Content-Type": "application/json",
+				},
+				body: []byte(`{"drivers" : [{"id": 1,"name": "driver-1"},{"id": 2,"name": "driver-2"}], "customer": {"id": 1111}}`),
+			},
+			expTransformedRequest: request{
+				headers: map[string]string{
+					"Content-Type": "application/json",
+				},
+				body: []byte(`{"instances": {"columns": ["rank", "driver_id", "customer_id", "driver_feature_1", "driver_feature_2", "driver_feature_3"], "data":[[0, 1, 1111, 1111, 2222, ["A", "B", "C"]], [1, 2, 1111, 3333, 4444, ["X", "Y", "Z"]]]}}`),
+			},
+			modelResponse: response{
+				headers: map[string]string{
+					"Content-Type": "application/json",
+				},
+				body:       []byte(`{"status": "ok"}`),
+				statusCode: 200,
+			},
+			expTransformedResponse: response{
+				headers: map[string]string{
+					"Content-Type": "application/json",
+				},
+				body:       []byte(`{"status": "ok"}`),
+				statusCode: 200,
+			},
+		},
+		{
+			name:         "table transformation with feast and series transformation",
+			specYamlPath: "../pipeline/testdata/valid_feast_series_transform.yaml",
+			mockFeasts: []mockFeast{
+				{
+					request: &feastSdk.OnlineFeaturesRequest{
+						Project: "default", // used as identifier for mocking. must match config
```

Comment:
Since we don't assert the request to feast, have you make sure it's as expected?

Reply:
Yeah, I manually checked it by logging the request here: https://github.com/gojek/merlin/pull/212/files/c9656a225bb7b0d9c3f59bac6f00d8be7c7c4686#diff-744e4800800a7662f4f3e134e1885cbd4571a9f3a802bbedeab46cf168251fe0R777

I don't like this approach though, we should be able to verify the request to the feast.